src/pages/EditQuestion.js

```diff
@@ -0,0 +1,98 @@
+import React, {useEffect, useState, useContext} from "react";
+import Typography from "@material-ui/core/Typography";
+import {makeStyles} from "@material-ui/core/styles";
+import Container from "@material-ui/core/Container";
+import Layout from "../layout/Layout";
+import Alert from "@material-ui/lab/Alert";
+import {FirebaseContext} from "../components/Firebase";
+import {useList} from "react-firebase-hooks/database";
+import Question from "../components/question/Question";
+
+const useStyles = makeStyles((theme) => ({
+  heroContent: {
+    padding: theme.spacing(8, 0, 6),
+  },
+  wrapper: {
+    margin: theme.spacing(1),
+    position: "relative",
+  },
+  buttonProgress: {
+    position: "absolute",
+    top: "50%",
+    right: "0",
+    marginTop: -12,
+    marginRight: 12,
+  },
+}));
+
+export default function EditQuestion() {
+  const classes = useStyles();
+
+  const firebase = useContext(FirebaseContext);
+
+  const [categories, loadingCategories, errorCategories] = useList(
+    firebase.categories()
```

Comment:
@emilymeroni we not using the firebase context anymore, here we can just import the api object from /api and replace:

```suggestion
    api.getCategories()
```